packages/react-examples/src/react-components/Migrations/utils.test.tsx

```diff
@@ -34,26 +35,48 @@ test('renders Markdown source blocks', () => {
   renderer.render(
     <CodeExample>
       {mockMDXSourceCodeBlock(`\`\`\`js
-                console.log("test");
-                \`\`\``)}
+        function test() {
```

Comment:
q: why this change?

Reply:
Wasn't testing multi-line code, current implementation didn't work properly for multiline.